.github/dependabot.yml

```diff
@@ -0,0 +1,25 @@
+# Basic dependabot.yml file with
+# minimum configuration for two package managers
+
+version: 2
+updates:
+  # Enable version updates for python
+  - package-ecosystem: "pip"
+    # Look for a `requirements` in the `root` directory
+    directory: "/"
```

Comment:
Won't this only modify the root requirements.txt? We need to tell it to modify all

Reply:
well, seems there is only one folder, so moving `./requirements.txt` to `./requirements/base.txt` and keeping the original `./requirements.txt` ad link

Reply:
SGTM!